db_stress_tool/db_stress_gflags.cc

```diff
@@ -789,7 +790,10 @@ DEFINE_int32(get_property_one_in, 1000,
 
 DEFINE_bool(sync_fault_injection, false,
             "If true, FaultInjectionTestFS will be used for write operations, "
-            " and unsynced data in DB will lost after crash.");
+            "and unsynced data in DB will lost after crash. In such a case we "
+            "track DB changes in a trace file in --expected_values_dir for "
```

Comment:
> Maybe give an example of the name of a trace file?

Done.

Reply:
> What is the expectation if expected_values_dir is empty but sync_fault_injection is set to true?

Empty `expected_values_dir` means no verification at startup, and verification at shutdown based only on the operations performed by that particular `db_stress` run. Combining it with `sync_fault_injection=true` would not bring additional coverage (compared to `sync_fault_injection=false`) because `sync_fault_injection` only causes unsynced data to be dropped at process end, which is after all verifications when `expected_values_dir` is empty. 

Since I believe the combination brings no additional coverage, it is currently not supported. The error looks like this:

```
$ ./db_stress  --expected_values_dir="" --sync_fault_injection=true
...
Error enabling history tracing: Not implemented:
```